Src/ILGPU/Runtime/OpenCL/CLStream.cs

```diff
@@ -28,6 +28,12 @@ public sealed class CLStream : AcceleratorStream
 
         private IntPtr queuePtr;
 
+        internal CLStream(Accelerator accelerator, IntPtr ptr)
```

Comment:
ISSUE: `CLStream` also needs the `responsible` boolean parameter, like `CudaStream`.

Reply:
Oops, forgot that.